.flake8

```diff
@@ -1,7 +1,7 @@
 [flake8]
 ignore =
     E121,E123,E126,E24,E704,W503,W504,  # these are ignored by default
-    E501,  # line too long
     E401,  # multiple imports
     E111,  # Indentation is not a multiple of 4
-    E114,  # Indentation is not a multiple of 4 comment
\ No newline at end of file
+    E114,  # Indentation is not a multiple of 4 comment
```

Comment:
Didn't we decide we didn't want to ignore this?

Reply:
Yeah I'll remove this.